examples/ibis/models/ibis_full_model_python.py

```diff
@@ -0,0 +1,41 @@
+import typing as t
+from datetime import datetime
+
+import ibis  # type: ignore
+import pandas as pd
+
+from sqlmesh import ExecutionContext, model
+from sqlmesh.core.model import FullKind
+
+
+@model(
+    "ibis.ibis_full_model_python",
+    kind=FullKind(),
+    columns={
+        "item_id": "int",
+        "num_orders": "int",
+    },
```

Comment:
could you write a description here explaining this is an ibis model that actually runs query and the other one is an ibis model that generates sql?